### PR TITLE
(FM-3134) Update changelog

### PR DESCRIPTION
**Pull request by ferventcoder**
Add the changes and the initial msutter releases to the
Changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,42 @@
-##2015-09-02 - Initial Release 0.8.0
+##2015-09-02 - Initial Puppet Labs Unsupported Release 0.8.0
+
+###Summary
+
+Initial release from Puppet Labs moving to PowerShell 5 to take advantage of Invoke-DSCResource. Update resource kit to latest and bundle the non-built in DSC Resources with the module.
```
Comment (Iristyle) [inline CHANGELOG.md]: `and bundle the DSC Resources from the PowerShell Gallery with the module.`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
+- Commit generated types / specs (MODULES-1956)
+- Vendor resource kit files and ensure on PSModulePath so user doesn't need to install all of the resources (FM-2652) (MODULES-2175)
+
+
+###Bugfixes
+- Fix `dsc_ensure => absent` (MODULES-2267)
+- Use original MOF gem for building types (MODULES-1957)
+- Call PowerShell with arguments (MODULES-2182)
+- Manage DSC resources without ensure (MODULES-2257)
+- Vendored resources should plugin sync (MODULES-2322)
```
Comment (Iristyle) [inline CHANGELOG.md]: Do we care about fixing a bug in the new feature (MODULES-2175) above?

Comment (ferventcoder) [inline CHANGELOG.md]: No, I'll remove that one.

File: CHANGELOG.md (hunk 1 above)
Comment (ferventcoder) [inline CHANGELOG.md]: Thanks, I wasn't cracked up about the wording.

**Pull request merged**